### DISCUSSION TITLE
Syntax for ValueList

### DISCUSSION THREAD

**Discussion by hchiba1** [pg-format/pg-formatter#15] [Ideas]
@nichtich proposed 
adding a syntax for multivalues to allow 
`1 :person name:Alice,Carol` as shortcut for 
`1 :person name:Alice name:Carol` both parsed into 
`{"id":"1","labels":["person"],"properties":{"name":["Alice","Carol"]}}`.

**Comment by hchiba1** [comment]
I understand that the PG format can be too redundant.

Currently, the 1st example above would be parsed into
`{"id":"1","labels":["person"],"properties":{"name":["Alice,Carol"]}}`.

So `,` should be treated as special in the PEG.

**Comment by hchiba1** [comment]
Modification itself was not so difficult (ab8849e).

It is easy to revert, if there is any problem.

**Comment by nichtich** [comment]
Seems done and accepted!
